lib/config/definitions.js

```diff
@@ -1397,7 +1397,13 @@ const options = [
     stage: 'package',
     type: 'json',
     default: {
-      fileMatch: ['\\.csproj$'],
+      fileMatch: ['(^|/)*\\.csproj$'],
+      nugetFeeds: [
```

Comment:
I think `registryUrls` is an existing concept we have that might fit this. Would these ever be "autodetected" from the repository, or users need to configure it manually in Renovate config?

Reply:
OK, here's what I'd like to do:

1. Do not use new `nugetFeeds` option (remove it)
2. Use existing `registryUrls` array instead
3. Do not explicitly define the default value inside `nuget` config. We can simply default to it inside datasource/nuget instead
4. Default to `v3`. But if you want to define a v2 feed then do it like `https://https://version2#v2`. The nuget datasource will look for this URL fragment and strip it before using it. Using `v3` should be optional.

Reply:
1-3 done and in 4 I made the auto detection system.